### ISSUE TITLE
Get status code from status page by using API

### ISSUE THREAD

**Issue by quadeare** [louislam/uptime-kuma#2605]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

Hi,

I would like to get a different HTTP code when we have an issue with one or multiple monitors.

For example, get a `/status` API path from status page.
- https://status.example.com/status/[page]/status

If all monitors are OK, so we get 200 HTTP code.
If we have one are multiple issues, maybe a 5xx error.

Thank you so much for your great job,
Have a nice day

### 🐻 Uptime-Kuma Version

1.19.4

### 💻 Operating System and Arch

Ubuntu 22.04

### 🌐 Browser

Google Chrome

### 🐋 Docker Version

RKE2 - k8s 1.24.4

**Comment by raulpcs** [comment]
I strongly agree with this request, i would like to have a way to monitor the general status of a monitor page though an API.
This way i can create widgets  on my personal website that tracks the status of the page.

**Comment by juzhiyuan** [comment]
+1 for this feature.

**Comment by SUNWUYUAN** [comment]
+1
